modules/core/src/main/scala/com.snowplowanalytics.snowplow.lakes/processing/LakeWriter.scala

```diff
@@ -128,14 +129,23 @@ object LakeWriter {
   /**
    * Implementation of the LakeWriter
    *
-   * The mutex is needed because we allow overlapping windows. They prevent two different windows
+   * The mutexes are needed because we allow overlapping windows. They prevent two different windows
    * from trying to run the same expensive operation at the same time.
+   *
+   * @param mutexForRemoteWriting
+   *   This mutex is needed because we allow overlapping windows. It prevents two different windows
+   *   from trying to run the same expensive operation at the same time
```

Comment:
This documentation is the exact same as the one above, would it be possible to be a bit more specific, like for `mutexForLocalAppending`?

Reply:
My mistake, I meant to delete the description at the top.  This description here is already specific to this mutex only.